packages/tools/src/launchEditor.ts

```diff
@@ -164,6 +216,56 @@ function findRootForFile(
   });
 }
 
+// On windows, find the editor executable path even if its not in the users path
+function editorWindowsLaunchPath(editor: string) {
+  if (fs.existsSync(editor)) {
+    // Editor is a full path to an exe, we can just launch it
+    return editor;
+  }
+
+  try {
+    execSync(`where ${editor}`, {stdio: 'ignore'});
+    // Editor is on the path, we can just launch it
+    return editor;
+  } catch (error) {
+    // ignore
+  }
+
+  try {
+    const editorNames = Object.values(COMMON_WINDOWS_EDITORS);
+    const editorImageNames = Object.keys(COMMON_WINDOWS_EDITORS);
+    for (let i = 0; i < editorNames.length; i++) {
+      const editorName = editorNames[i];
+      if (editor.toLowerCase() === editorName.toLowerCase()) {
```

Comment:
Do we also need case insensitivity when guessing the editor?

Reply:
I dont think so.  The case there is based on the actual case of the executable.  So unless someone went and renamed Code.exe to code.exe, (why would someone do that?) then it should match.  This one it makes sense to leave as case insensitive since someone could have set REACT_EDITOR=Code or REACT_EDITOR=code and this would go look up the correct executable path for it if it isn't already in the users path.